2227/2/lib/date_analyzator.rb

```diff
@@ -0,0 +1,9 @@
+# Date Analysis
+class DateAnalyzer
```

Comment:
and this name is too abstract. What about MostObsceneRappersFinder or something like this.